src/operator/quantization/quantize_graph_pass.cc

```diff
@@ -117,9 +117,10 @@ inline bool NeedQuantize(NodePtr node, const std::unordered_set<std::string>& ex
 }
 
 Graph QuantizeGraph(Graph &&src) {
-  static auto& quantized_op_map = Op::GetAttr<mxnet::FQuantizedOp>("FQuantizedOp");
-  static auto& need_requantize_map = Op::GetAttr<mxnet::FNeedRequantize>("FNeedRequantize");
-  static auto& avoid_quantize_input_map =
+  static const auto& flist_outputs = nnvm::Op::GetAttr<nnvm::FListOutputNames>("FListOutputNames");
```

Comment:
nit: output_list is a better

Reply:
Just want to keep the same name as others in this file, like [#L326](https://github.com/apache/incubator-mxnet/blob/master/src/operator/quantization/quantize_graph_pass.cc#L326).
Anyway, this could be changed as you mentioned.

Reply:
If so, keeping it is fine with me as long as it is consistent.